app/code/scout/twitter.ts

```diff
@@ -32,7 +32,48 @@ export async function getUserData(username: string) {
   try {
     const response = await axios.get(getUserEndpoint, config);
     return response.data;
-  } catch (error:any) {
+  } catch (error: any) {
     console.error(`Error fetching user data: ${error.message}`);
   }
 }
+
+export async function fetchTwitterData(twitterUsername: string): Promise<any> {
+  const options = {
+    headers: {
+      Authorization: `Bearer ${TWITTER_BEARER_TOKEN}`
+    }
+  };
+  const getUserEndpoint =
+    "https://api.twitter.com/2/users/by?usernames=" + twitterUsername;
+  const getUserResponse1 = await fetch(getUserEndpoint, options);
+  const getUserResponseJson = await getUserResponse1.json();
+  const userId = getUserResponseJson.data[0].id;
+  const getTweetsEndpoint = `https://api.twitter.com/2/users/${userId}/tweets?tweet.fields=created_at&max_results=5`;
+  console.log("\n= = = = = SENDING TWITTER REQUEST = = = = = \n");
+  const getTweetsResponse = await fetch(getTweetsEndpoint, options);
+  const getTweetsData = await getTweetsResponse.json();
+  const lastTweet = getTweetsData.data[getTweetsData.data.length - 1];
```

Comment:
we want the LATEST tweet, aka the newest :P  I assume they come in the order of newest first?

Reply:
Ah sorry about that. This is now fixed.